src/utils.h

```diff
@@ -126,3 +127,21 @@ inline int getRandomInt(int min, int max)
 
     return distrib(gen);
 }
+
+inline std::chrono::time_point<std::chrono::high_resolution_clock> highResolutionNow()
+{
+    return std::chrono::high_resolution_clock::now();
+}
+
+inline long double elapsedMicroseconds(std::chrono::time_point<std::chrono::high_resolution_clock> startTime)
+{
+    return std::chrono::duration<long double, std::micro>(highResolutionNow() - startTime).count();
+}
+
+bool fileExists(const char *filename)
+{
+    struct stat dummy;
+
+    // Use stat to keep compatibility with 7th Heaven
+	return stat(filename, &dummy) == 0;

```

Comment:
Can you fix the indentation here please? :D

Reply:
sure